src/modules/map/components/olMap/olStyleUtils.js

```diff
@@ -212,31 +213,38 @@ export const measureStyleFunction = (feature) => {
 	return styles;
 };
 
-export const modifyStyleFunction = () => {
+export const modifyStyleFunction = (feature) => {
+	const getParentFeature = (child) => {
+		const features = child.get('features');
+		return features[0] ? features[0] : null;
+	};
+	const currentFeature = feature ? getParentFeature(feature) : null;
+	const color = currentFeature ? getColorFrom(currentFeature) : Red_Color;
+
 	return [new Style({
 		image: new CircleStyle({
-			radius: 8,
+			radius: 6,
 			stroke: new Stroke({
-				color: Red_Color,
-				width: 1
+				color: White_Color,
+				width: 2
 			}),
 			fill: new Fill({
-				color: White_Color
+				color: color
```

Comment:
Would suggest adding real test for all style definitions like in `highlight/styleUtils.test.js`
`expect(styles).toBeTruthy()`  =>  `expect(styles).toEqual([style]);`